extensions/nepali-date-converter/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "nepali-date-converter",
+  "title": "Nepali Date Converter",
+  "description": "It converts current, provided date to Nepali date",
+  "icon": "extension_icon.png",
+  "author": "beedhan",
+  "categories": [
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Convert to BS",
```

Comment:
Is there a name that suits better?

Reply:
It should have been B.S. which is an abbreviation of Bikram Samwat. I think "Convert to Nepali Date" or "Convert to Nepali Calendar" would be better.